src/util.ts

```diff
@@ -76,6 +76,8 @@ export async function synthApp(command: string, outdir: string, stdout: boolean,
 
   if (!found) {
     console.error('No manifests synthesized');
+  } else {
+    console.log('Manifests synthesized!')
```

Comment:
I don't think this necessary though. If we found manifests, then we [print their path](https://github.com/cdk8s-team/cdk8s-cli/blob/2.x/src/util.ts#L71) already, indicating a successful synthesis.